pureconfig/README.md

```diff
@@ -0,0 +1,17 @@
+# Toggle
+This type class is basically an Option with the possibility to keep the whole configuration in place and change a special boolean flag only. Please check the example below:
```

Comment:
I don't think `Toggle` is a type class (as in the pattern), it's either a type or a class.

Reply:
Also this whole sentence does not seem very clear. The example makes it clear but please try to describe why this exists and how it can be used (I think the relation to `Option` might not be important too much).

Reply:
Please let me know whether this one is more understandable or not:
https://github.com/avast/scala-server-toolkit/pull/982/files#diff-f197acdd0685d1ec179f803054d18d6cb451bb451d20901c4f5d33c196a71df0R69

Reply:
Much better! Thanks.